### ISSUE TITLE
Vanish break align-tabsuffix

### ISSUE THREAD

**Issue by WarnDa** [NEZNAMY/TAB#604]
### Server version

Pufferfish 1.17.1

### TAB version

3.1.1

### Describe the bug

When a player vanish (using essentialsX vanish), sometimes it will break the "align-tabsuffix-on-the-right: true"

### Expected behavior

It should still align the suffix on the right

### Steps to reproduce

Step 1 : setup a suffix
Step 2 : align-tabsuffix-on-the-right: true
Step 3 : vanish for X time
Step 4 : even when unvanished, your suffix it still not at right

### Additional info

![image](https://user-images.githubusercontent.com/31826333/169660483-599b8e71-4f68-4488-9ede-b1d0753ef251.png)

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform

**Comment by NEZNAMY** [comment]
Did you give the people who can see vanished players `tab.seevanished` permission?

**Comment by WarnDa** [comment]
> tab.seevanished

No, but on the screen we are both unvanished (but we vanished, and while we were vanished it broke the suffix)

**Comment by NEZNAMY** [comment]
Give them the permission and try again.

**Comment by WarnDa** [comment]
I got tab.seevanished (as opped and * permission) and for me rogan is also bugged

**Comment by WarnDa** [comment]
Not possible to get debug, or any info to help you find the issue ? :c

**Comment by NEZNAMY** [comment]
Set up a test server with only TAB and vanish plugin and create & follow exact steps.

**Comment by WarnDa** [comment]
Too hard to find a way to reproduce...
So I can't find an exact way to reproduce it
